examples/rosettacode/rock_paper_scissor.nit

```diff
@@ -0,0 +1,48 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+#
+# Task: Programming_Task/Rock-paper-scissors
+# SEE: <http://rosettacode.org/wiki/Rock-paper-scissors>
+#
+#author: Istvan szalai 22/05/2015
+
+fun core(player, computer: Int): Int do
+	if player == computer then return 0
+	if player > computer then return 1
+	return 2
+
+end
+
+fun resultMessage(number: Int): String do
+	if number == 0 then return "it's a tie !"
+	if number == 1 then return "You win"
+	return "You loose"
+end
+
+fun playMessage(number: Int): String do
+	if number == 5 then return "The computer played rock;"
+	if number == 6 then return "the computer played scissors;"
+	return "computer played paper;"
+end
+
+fun convert(std: String): Int do
+	if std == "rock" then return 5
+	if std == "scissors" then return 6
+	return 7
+end
+
+fun computer: Int do return 3.rand + 5
+
+var x = ""
+var gameLoop = true
+var test = false
+while gameLoop do
```

Comment:
Instead of a while true like this, use a `loop end` structure, especially since gameLoop is never used